### ISSUE TITLE
Installing on macOS Big Sur

### ISSUE THREAD

**Issue by rouson** [Fortran-FOSS-Programmers/ford#351]
Does FORD 6.1.1, released on July 20, contain the fix for issue #326, which was fixed on July 14?  If so, what are the recommended installation options for 6.1.1 on macOS Big Sur?  I tried the following commands from FORD's top-level README.md:
```
brew update
brew install FORD
```
after which I get

ford --version 
FORD, version 6.0.0
```
I also tried `brew install --HEAD ford`, but still ended up `ford --version` returning 6.0.0.

Also, `brew install python` doesn't install `pip` so `pip install ford` doesn't work anymore either.  Please update the README.md with new installation instructions.

**Comment by ZedThree** [comment]
Hi @rouson, yep, FORD 6.1.1 does contain that fix.

It looks like the brew formula is out of date. From the looks of things, we need to update:

https://github.com/Homebrew/homebrew-core/blob/HEAD/Formula/ford.rb

following the instructions here:

https://docs.brew.sh/How-To-Open-a-Homebrew-Pull-Request

If anyone has any experience with brew who would like to do this, that would be much appreciated :)
Otherwise, I will try to look at this myself ASAP

**Comment by ZedThree** [comment]
I think this is now fixed and 6.1.6 is now installable via brew thanks to @chenrui333 !
